### PR TITLE
facter. avoid admins in users count

### PR DESCRIPTION
**Pull request by edospadoni**
Nethserver/dev#6051

### CODE REVIEW

File: root/opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/webtop.rb [hunk 1]
```diff
@@ -7,7 +7,15 @@
         webtop = {}
         pass = File.read("/var/lib/nethserver/secrets/webtop5").strip
 
-        for t in ['core.users', 'core.shares', 'calendar.calendars', 'contacts.contacts'] do
+        query_users = Facter::Core::Execution.exec("psql 'postgresql://sonicle:#{pass}@localhost:5432/webtop5' -Aqt -c \"select user_id from core.users where type = 'U' and user_id != 'admin'\"")
+        actual_users = query_users.split("\n")
+
+        query_provider = Facter::Core::Execution.exec("echo '{\"action\":\"list-users\"}' | /usr/bin/sudo /usr/libexec/nethserver/api/system-users/read | jq -r 'keys | .[]'")
```
Comment (gsanchietti) [inline root/opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/webtop.rb]: Call to `sudo` is not required here.

Eventually, you could also call directly `/usr/libexec/nethserver/list-users -s` helper, but this is a matter of taste

File: root/opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/webtop.rb [hunk 2]
```diff
@@ -7,7 +7,15 @@
...
+        provider_user = query_provider.split("\n")
+
+        webtop['users'] = (actual_users & provider_user).length
```
Comment (gsanchietti) [inline root/opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/webtop.rb]: This is cool! :)

File: root/opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/webtop.rb (hunk 1 above)
Comment (edospadoni) [reply root/opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/webtop.rb]: Changed

File: root/opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/webtop.rb (hunk 2 above)
Comment (edospadoni) [reply root/opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/webtop.rb]: ruby magic! 🧙

**Pull request merged**